prime-dotnet-webapi/Configuration/Database/Resources/pharmacy-tech-ru-no-obo-toa-v2.html

```diff
@@ -0,0 +1,651 @@
+          or other decision-maker, to be invalid, this Agreement will be interpreted as if
+          such provisions were not included.
+        </p>
+
+      </li>
+      <li>
+
+        <p>
+          <strong>Survival.</strong> Any provision of this Agreement that expressly or by its nature
+          continues after termination, shall survive termination of this Agreement.
+        </p>
+
+      </li>
+      <li>
+
+        <p>
+          <strong>Governing Law.</strong> This Agreement will be governed by and will be construed and
+          interpreted in accordance with the laws of British Columbia and the laws of
+          Canada applicable therein.
+        </p>
+
+      </li>
+      <li>
+
+        <p>
+          <strong>Assignment Restricted.</strong> Your rights and obligations under this Agreement may
+          not be assigned without the prior written approval of the Province.
+        </p>
+
+      </li>
+      <li>
+
+        <p>
+          <strong>Waiver.</strong> The failure of the Province at any time to insist on performance of any
+          provision of this Agreement by you is not a waiver of its right subsequently to
+          insist on performance of that or any other provision of this Agreement.
+        </p>
+
+      </li>
+      <li>
+
+        <p>
+          <strong>Province May Modify this Agreement.</strong> The Province may amend this
+          Agreement, including this section, at any time in its sole discretion:
+        </p>
+
+        <ol type="i">
+          <li>
+            by written notice to you, in which case the amendment will become
+            effective upon the later of (A) the date notice of the amendment is first
+            delivered to you, or (B) the effective date of the amendment specified by
+            the Province, if any; or
+          </li>
+          <li>
+            by publishing notice of any such amendment in the PharmaCare
+            Newsletter, in which case the notice will specify the effective date of the
+            amendment, which date will be at least 30 (thirty) days after the date that
+            the PharmaCare Newsletter containing the notice is first published.
+          </li>
+        </ol>
+
+        <p>
+          If you use PharmaNet after the effective date of an amendment described in (i) or
+          (ii) above, you will be deemed to have accepted the corresponding amendment,
+          and this Agreement will be deemed to have been so amended as of the effective
+          date. If you do not agree with any amendment for which notice has been provided
+          by the Province in accordance with (i) or (ii) above, you must promptly (and in
+          any event before the effective date) cease all access or use of PharmaNet by yourself and take the steps
+          necessary to terminate this Agreement in accordance
+          with section 10.
+        </p>
+
+      </li>
+    </ol>
+
+  </li>
+</ol>
+<p>
+  Version date: January 28, 2022
```

Comment:
Is this date on the other TOAs?

Reply:
Yes, it's at the bottom of the word file:
![image](https://user-images.githubusercontent.com/34261366/157497477-8a620f41-c69a-44d0-bb8b-70a04014953e.png)

Reply:
Yes please remove it. @AMiKaH